packages/backend/src/projectAdapters/dbtBaseProjectAdapter.ts

```diff
@@ -76,18 +78,20 @@ export class DbtBaseProjectAdapter implements ProjectAdapter {
         }
         Logger.debug('Get dbt manifest');
         const { manifest } = await this.dbtClient.getDbtManifest();
-
         // Type of the target warehouse
         if (!isSupportedDbtAdapter(manifest.metadata)) {
             throw new ParseError(
                 `Dbt project not supported. Lightdash does not support adapter ${manifest.metadata.adapter_type}`,
                 {},
             );
         }
+
+        const manifestModels = getModelsFromManifest(manifest);
+        const compiledModels = getCompiledModels(manifestModels, undefined);
```

Comment:
This code seems to be working with filters, but we're missing the `maybeCompileModelsAndJoins` , which requires some extra calls to dbt. it would be nice if we can skip this for now. Let me know if you think this is a big blocker.